dmorg.tex

```diff
@@ -22,7 +22,7 @@ \section{Data Management Organizational Structure}
 As a diverse and distributed organization DM staff will participate in a considerable number of meetings.
 NSF and Aura have many rules on meeting attendance and LSST keep policies updated accordingly in \citeds{LPM-191} and \citeds{Document-13760}. This includes the travel summary report template \citedsp{Document-13762} every traveler must fill after attending a meeting.
 
-The DMLT (\secref{sect:dmlt}) may, on occasion, require that travellers to a specific meeting of direct interest provide a detailed debriefing note or presentation.
+The DMLT (\secref{sect:dmlt}) may, on occasion, require that travelers to a specific meeting of direct interest provide a detailed debriefing note or presentation.
```

Comment:
Do we need to make this change? My dictionary says that either one or two “l”s is acceptable in American English.